L1Trigger/CSCTriggerPrimitives/python/cscTriggerPrimitiveDigis_cfi.py

```diff
@@ -1,16 +1,21 @@
 import FWCore.ParameterSet.Config as cms
-
 from L1Trigger.CSCCommonTrigger.CSCCommonTrigger_cfi import *
-# Default parameters for CSCTriggerPrimitives generator
-# =====================================================
-cscTriggerPrimitiveDigis = cms.EDProducer("CSCTriggerPrimitivesProducer",
+from L1Trigger.CSCTriggerPrimitives.alctParams_cfi import *
```

Comment:
Perhaps it is just matter of taste, but I would prefer to write explicitly the modules that you are importing.